nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -124,26 +124,23 @@ in
             "~ ^(\\/[\\d]+\\.[\\d]+\\.[\\d]+[\\.|-][\\w]+)?\\/(document_editor_service_worker\\.js)$".extraConfig =
               ''
                 expires 365d;
-                # gzip_static on;
```

Comment:
I kept this comment to keep this code as similar to upstream's `/etc/nginx/includes/ds-docservice.conf` as possible, to make future diffs/updates easier. I'm OK with removing it, I suppose, though.

Reply:
They have those commented, too 💀 sigh

Reply:
I am not a fan of keeping commented code with no action attached to it.